src/flow-spec/IFlow.ts

```diff
@@ -34,6 +34,8 @@ export interface IFlow {
 
   /**
    * A human-readable name for the Flow content
+   *
+   * @pattern ^[a-zA-Z_]\w*$
```

Comment:
Does this mean if I'm a french user, I can't use characters that have accents in them?

Reply:
Good spot @AlteaDown , I didn't thought about that. To force validation for now, we can use French string if we start with non `[a-zA-Z_]` chars.

@bzabos , I think this remark concerns almost all human required string props (block.name, etc). Any thoughts how should we consider it in the future ?

Reply:
@safydy is this change solely to get the generated json schema to validate this as a required field? 

If so, I am wondering if there's a way to say "min length = 3 characters" or something like that instead?

Reply:
@bzabos , the change is to use the same pattern as from IBlock.name, IExit.tag,  https://github.com/FLOIP/flow-runner/blob/master/dist/resources/flowSpecJsonSchema.json#L26
If I'm not wrong, the idea behind is to give a value which could be used as `variable name` later.
> A human-readable "variable name" ... This must be restricted to word characters so that it can be used as a variable name in expressions. When blocks write results output, they write to a variable corresponding the name of the block.

The current regex is forcing us to have a name with a leading word. (eg: `e123` is valid, `2lorem` is not valid)

Reply:
@bzabos , I saw [you saying](https://votomobile.slack.com/archives/CMQDVRDN3/p1623960055046300?thread_ts=1623960049.046200&cid=CMQDVRDN3) 
> `flow.name` is free form text (not localizable)
meaning it's not suitable as variable name

In that case, your last suggestion `"min length = 3 characters"` makes sense. I will update this pattern for now.

Reply:
Oh, awesome, thanks @safydy! 😃